test/performance/benchmarks/load-test/continuous/main.go

```diff
@@ -133,8 +133,9 @@ func main() {
 
 	q.Input.ThresholdInputs = append(q.Input.ThresholdInputs,
 		newLoadTest95PercentileLatency(tbcTag),
-		newLoadTestMaximumLatency(tbcTag),
-		newLoadTestMaximumErrorRate(tbcTag))
+		newLoadTestMaximumLatency(tbcTag))
+	// Temporarily disable the error rate check before https://github.com/knative/serving/issues/10074 is fixed.
```

Comment:
No comments (except mine) since early November.  What's up with this?  @ZhiminXiang

Reply:
The fix is in Istio 1.9. Will check with Istio team and see if it can be backported to earlier Istio version:
https://github.com/knative/serving/issues/10074#issuecomment-771076659

Reply:
Shall we merge this PR for now to avoid more spams being sent to Slack?

/retest

Reply:
/cc @vagababov

Reply:
Can you add an issue+todo  so we don't forget to fix it eventually

Reply:
Done.